src/modules/launcher/Wox.Core/Plugin/PluginManager.cs

```diff
@@ -194,13 +194,14 @@ private static List<Result> UpdateResultWithActionKeyword(List<Result> results,
         {
             foreach (Result result in results)
             {
-                if (!string.IsNullOrEmpty(result.QueryTextDisplay))
+                if (string.IsNullOrEmpty(result.QueryTextDisplay))
```

Comment:
Is the behavior supposed to be that if it is not-empy it should use the initial value for QueryTextDisplay? Where does that get set?

Reply:
`QueryTextDisplay` is the text to be displayed in querytextbox when a result is selected using keyboard navigation. `QueryTextDisplay` is set by plugins in case they have something specific to be displayed. In case plugins do not set it, we use the title.